lib/rules/require-curly-braces.js

```diff
@@ -112,30 +112,30 @@ module.exports.prototype = {
                     return;
                 }
 
-                if (node.test === null && typeIndex['default']) {
+                if (node.test === null && typeIndex.default) {
                     addError('Default', node);
                 }
 
-                if (node.test !== null && typeIndex['case']) {
+                if (node.test !== null && typeIndex.case) {
```

Comment:
IE8 and older doesn't support it. Yep ;-\
Should we set `{es3: true}` in our `.jscsrc`?